calculator.js

```diff
@@ -1,5 +1,54 @@
 const exampleAdditionInput = {
   num1: 3,
-  num2: 5,
-  operation: 'add',
+  num2: 1,
+  operation: '/',
 }
+
+//another way of writing it ...const add = (num1, num2) num1 + num2;
+const add = function(num1, num2) {
+  return num1 + num2;
+}
+
+const subtract = function(num1, num2){
+  return num1 - num2;
+}
+
+const divide = function(num1, num2){
+  return num1 / num2;
+}
+
+const multiply = function(num1, num2){
+  return num1 * num2;
+}
+const validOperations = ['add', '+', 'subtract', '-', 'divide', '/', 'multiply', '*']
+
+
+
+const calculate = function(input){
+  if (validOperations.includes(input.operation)){
+    if (Number.isInteger(input.num1) && Number.isInteger(input.num2)){    
+    // check to see what the input is and depending on that call for the helper methods
+      if (input.operation === 'add' || input.operation === '+'){
+        return add(input.num1, input.num2);
+      }
+      else if (input.operation === 'subtract' || input.operation === '-'){
+        return subtract(input.num1, input.num2);
+      }
+      else if (input.operation === 'divide' || input.operation === '/'){
+        return divide(input.num1, input.num2);
+      }
+      else if (input.operation === 'multiply' || input.operation === '*'){
+        return multiply(input.num1, input.num2);
+      }
```

Comment:
Since you mentioned you wanted to get it refactored into a `switch`:

```suggestion
      switch(input.operation) {
        case 'add':
        case '+':
          return add(input.num1, input.num2);
        case 'subtract':
        case '-':
          return subtract(input.num1, input.num2);
        case 'divide':
        case '/':
          return divide(input.num1, input.num2);
        case 'multiply':
        case '*':
          return multiply(input.num1, input.num2);
      }
```